dominion/tasks.py

```diff
@@ -155,6 +165,9 @@ def build(user_id, image):
     if env.get('OS', None):
         env['BASE_DIR'] = os.path.join(APP.conf.get('BASE_SYSTEMS'), env['OS'])
 
+    if env['XFCE4'] == 'true':
+        env['BASE_DIR'] = os.path.join(APP.conf.get('BASE_SYSTEMS'), env['OS'] + '-xfce4')
```

Comment:
@leocoan I suggest to refactor `env['OS'] + '-xfce4'` to a method called get_chroot_name or something like that. See `routines.get_os_name` as a reference.

Reply:
Okay, let's think about it when add another desktop environment